server/routes/reviews.js

```diff
@@ -11,13 +11,12 @@ reviews
     if (req.params.sort_order) {
```

Comment:
Line 8 & 10 - sneaky little var here!

Reply:
> Chef's kiss.
> 
> Approving but maybe change the var in my comment above when you get a chance

Gracias señor. What don’t you like about my little vars? I thought it was a nice way to handle optional parameters (sort order is optional, at least)

Reply:
Ah si, ever since es6, i believe var is usually avoided at all costs. Although functional, the function scope of var can lead to undesired scope pollution, so it has become convention that const and let are to be used as they follow the stricter block scope

Reply:
Ah gotcha. Noted—will pick up in next round of changes